internal/pkg/agent/cmd/install.go

```diff
@@ -61,6 +66,13 @@ would like the Agent to operate.
 	cmd.Flags().Bool(flagInstallDevelopment, false, "Install into a standardized development namespace, may enable development specific options. Allows multiple Elastic Agents to be installed at once. (experimental)")
 	_ = cmd.Flags().MarkHidden(flagInstallDevelopment) // For internal use only.
 
+	// Active directory user specification
+	cmd.Flags().String(flagInstallCustomUser, "", "Custom user used to run Elastic Agent")
+	cmd.Flags().String(flagInstallCustomGroup, "", "Custom group used to access Elastic Agent files")
+	if runtime.GOOS == "windows" {
+		cmd.Flags().String(flagInstallCustomPass, "", "Password for Active directory user used to run Elastic Agent")
```

Comment:
On Windows a password is required for a user event if its a custom user that is not in Active Directory.

```suggestion
		cmd.Flags().String(flagInstallCustomPass, "", "Password for user used to run Elastic Agent")
```